unity/Assets/Maroon/scenes/experiments/OpticsSimulations/Scripts/Manager/PresetManager.cs

```diff
@@ -0,0 +1,361 @@
+﻿using System.Collections.Generic;
+using Maroon.scenes.experiments.OpticsSimulations.Scripts.Camera;
+using Maroon.scenes.experiments.OpticsSimulations.Scripts.TableObject.LightComponent;
+using Maroon.scenes.experiments.OpticsSimulations.Scripts.TableObject.OpticalComponent;
+using Maroon.scenes.experiments.OpticsSimulations.Scripts.Util;
+using UnityEngine;
+
+namespace Maroon.scenes.experiments.OpticsSimulations.Scripts.Manager
+{
+    public class PresetManager : MonoBehaviour
+    {
+        public static PresetManager Instance;
+        
+        [Header("Prefabs: Light Sources")]
+        [SerializeField] private LaserPointer laserPointer;
+        [SerializeField] private ParallelSource parallelSource;
+        [SerializeField] private PointSource pointSource;
+        
+        [Header("Prefabs: Optical Components")]
+        [SerializeField] private Aperture aperture;
+        [SerializeField] private Eye eye;
+        [SerializeField] private Lens lens;
+        [SerializeField] private TableObject.OpticalComponent.Mirror mirror;
+
+        [Header("Camera")] 
+        [SerializeField] private GameObject mainCamera;
+        private CameraControls _camControls;
+
+        private OpticalComponentManager _ocm;
+        private LightComponentManager _lcm;
+        private UIManager _uim;
+        private ExperimentManager _em;
+
+        private void Awake()
+        {
+            if (Instance == null)
+                Instance = this;
+            else
+            {
+                Debug.LogError("SHOULD NOT OCCUR - Destroyed PresetManager");
+                Destroy(gameObject);
+            }
+        }
+        
+        public void Start()
+        {
+            _ocm = OpticalComponentManager.Instance;
+            _lcm = LightComponentManager.Instance;
+            _uim = UIManager.Instance;
+            _em = ExperimentManager.Instance;
+            _camControls = mainCamera.GetComponent<CameraControls>();
+            
+            LensAndMirror();
+        }
+
+        public void TablePresets(int nr)
+        {
+            switch (nr)
+            {
+                case 0: return;
+                case 1: LensAndMirror(); break;
+                case 2: FocalLength(); break;
+                case 3: StandardEye(); break;
+                case 4: NearsightedEye(); break;
+                case 5: FarsightedEye(); break;
+                // case 6: UnderwaterVision(); break;
+                case 6: MagnifyingGlass(); break;
+                case 7: KeplerianTelescope(); break;
+                case 8: GalileanTelescope(); break;
+                case 9: NewtonianTelescope(); break;
+                case 10: Microscope(); break;
+                case 11: LightEmittingDiode(); break;
+                case 12: OpticalFiber(); break;
+            }
+        }
+        
+        private void LensAndMirror()
```

Comment:
later we could load these settings from a json file or something similar. also to support loading settings from Peters website via the WebGL receiver class.